openid-federation-1_0.xml

```diff
@@ -1787,6 +1802,11 @@
               OPTIONAL. URL of the documentation of
               conditions and policies relevant to this Entity.
             </t>
+            <t hangText="information_uri">
```

Comment:
This somewhat overlaps with the meaning of `resource_documentation` at https://www.iana.org/assignments/oauth-parameters/oauth-parameters.xhtml#protected-resource-metadata, but as in the previous case, I think this is acceptable.